descriptions/object_types_information.md

```diff
@@ -0,0 +1,33 @@
+This structure defines a collection of all registered types of kernel objects.
+
+# Applicable to
+ - `NtQueryObject` with `ObjectTypesInformation`.
+
+# Members
+
+## NumberOfTypes
+The total number of types present.
+
+## (Variable part)
+The variable part of the structure contains variable-sized `OBJECT_TYPES_INFORMATION` structures following each other. Each entry occupies `sizeof(OBJECT_TYPE_INFORMATION) + entry->TypeName.MaximumLength` bytes and is aligned up to a pointer boundary.
```

Comment:
You likely meant "contains variable-sized `OBJECT_TYPE_INFORMATION` structures" here.

Reply:
Yes, that's surely a typo

Reply:
👍 pushed a fix